lib/request_headers_logger/message_queue/message_queue_plugin.rb

```diff
@@ -25,18 +25,19 @@ def symbolize(obj = {})
         store = symbolize(properties.headers).dig(:store) || {}
         RequestHeadersMiddleware.store = store
 
-        set_dj_loggers
+        set_mq_loggers
       end
 
       lifecycle.after(:consume) do |delivery_info, properties, payload|
         RequestHeadersMiddleware.store = {}
       end
     end
 
-    def self.set_dj_loggers
+    def self.set_mq_loggers
       RequestHeadersLogger.configure do |config|
-        config[:loggers] = [MessageQueue.logger]
+        config[:loggers] << MessageQueue.logger
```

Comment:
memory leak if I read the code correctly (this code hits on every message delivery right?)